package.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "morpho-seatbelt",
+    "version": "1.0.0",
+    "description": "Testing suite to verify Dao's setUp",
+    "author": "Morpho Labs <security@morpho.xyz>",
+    "license": "MIT",
+    "keywords": [
+      "defi",
+      "protocol",
+      "smart-contracts",
+      "DAO"
+    ],
+    "homepage": "https://github.com/morpho-org/morpho-seatbelt/blob/main/README.md",
+    "bugs": {
+      "email": "security@morpho.xyz"
+    },
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/morpho-org/morpho-seatbelt"
+    },
+    "scripts": {
+      "test": "forge test",
+      "postinstall": "husky install"
+    },
+    "devDependencies": {
+      "@commitlint/cli": "^17.6.3",
+      "@commitlint/config-conventional": "^17.6.3",
+      "husky": "^8.0.3",
+      "lint-staged": "^13.2.2",
+      "prettier": "^2.8.8"
+    },
+    "lint-staged": {
+      "*.sol": "forge fmt",
+      "*.json": "prettier"
+    },
+    "commitlint": {
+      "extends": [
+        "@commitlint/config-conventional"
+      ]
+    }
```

Comment:
This is not used if we're not enforcing conventional commit msg. To enforce it, we should use the commit-msg hook